### DISCUSSION TITLE
quarkus-roq/posts/write-your-blog-posts-in-asciidoc/

### DISCUSSION THREAD

**Discussion by giscus** [quarkiverse/quarkus-roq#216] [Comments]
# quarkus-roq/posts/write-your-blog-posts-in-asciidoc/

Automatically generate html from AsciiDoc content

https://pages.quarkiverse.io/quarkus-roq/posts/write-your-blog-posts-in-asciidoc/

**Comment by melloware** [comment]
This is an awesome feature!

**Comment by ia3andy** [comment]
@jtama @mcruzdev Thanks for this, I think many people prefer asciidoc to md and will love this. 

You Roq!

**Reply by melloware** [reply]
I think both Giscus and AsciiDoc are so slick.

**Reply by jtama** [reply]
Giscus is a great add!

**Reply by mcruzdev** [reply]
Thanks @RayzaAnchayhua for adding Giscus!
